Understanding Ownership/Move Semantics/Mutable Argument/task.md

```diff
@@ -1,6 +1,6 @@
 ## Mutable Argument
 
-Make me compile without adding new lines –– just changing existing lines! (no lines with multiple semicolons necessary!)
+Make me compile without adding new lines — just changing existing lines! (no lines with multiple semicolons necessary!)
 
 <div class="hint">
   The difference between this one and the previous ones is that the first line of `fn fill_vec` that had `let mut vec = vec;` is no longer there.
```

Comment:
Unlike the previous cases, the first line of `fn fill_vec`, which had `let mut vec = vec;`, is now gone.